### PR TITLE
 Added new port configurations for buffered payload types. 🚀

### PR DESCRIPTION
**Pull request by michaelbeutler**
💪 This PR introduces some exciting new enhancements to our payload configurations! 💥

### Ports 🚧
🔥 Here are the ports that have been added or updated in this PR:

- **Port 10**: 🆕 Updated GNSS uplink including Unix timestamp
- **Port 50**: 🔄 Combined GNSS and Wi-Fi uplink
- **Port 105**: 📥 Buffered Wi-Fi uplink
- **Port 110**: 📥 Buffered updated GNSS uplink
- **Port 150**: 📥 Buffered combined GNSS and Wi-Fi uplink

🚀 These new port configurations enable us to handle different types of buffered payload data more efficiently and effectively. By incorporating the Unix timestamp in Port 10, we ensure accurate time data for our GNSS uplink. Additionally, combining GNSS and Wi-Fi uplink in Port 50 provides a seamless and comprehensive solution.

🔧 The addition of buffered ports 105, 110, and 150 enhances our ability to handle and store Wi-Fi and GNSS data, optimizing the transmission and retrieval process.

🎉 Let's merge this PR and leverage these awesome new port configurations to improve our payload handling capabilities! 🎉

### CODE REVIEW

**Comment by michaelbeutler** [comment]
@havar-miromico, I encountered some issues with the payload structure for Port 105 during testing. Here are the details:

Payload: `000500e0286d8a9478a9f0b0140c96bbc3726c9a74b58da7`

Configurations used:
```go
decoder.PayloadConfig{
  Fields: []decoder.FieldConfig{
    {Name: "BufferLevel", Start: 0, Length: 2},
    {Name: "Timestamp", Start: 3, Length: 4},
    {Name: "Moving", Start: 7, Length: 1},
    {Name: "Mac1", Start: 8, Length: 6, Optional: true},
    {Name: "Rssi1", Start: 14, Length: 1, Optional: true},
    {Name: "Mac2", Start: 15, Length: 6, Optional: true},
    {Name: "Rssi2", Start: 21, Length: 1, Optional: true},
    {Name: "Mac3", Start: 22, Length: 6, Optional: true},
    {Name: "Rssi3", Start: 28, Length: 1, Optional: true},
    {Name: "Mac4", Start: 29, Length: 6, Optional: true},
    {Name: "Rssi4", Start: 35, Length: 1, Optional: true},
    {Name: "Mac5", Start: 36, Length: 6, Optional: true},
    {Name: "Rssi5", Start: 42, Length: 1, Optional: true},
    {Name: "Mac6", Start: 43, Length: 6, Optional: true},
    {Name: "Rssi6", Start: 49, Length: 1, Optional: true},
  },
  TargetType: reflect.TypeOf(Port105Payload{}),
}
```

Output Received:
```json
{
  "bufferLevel": 5,
  "mac1": "78a9f0b0140c",
  "mac2": "bbc3726c9a74",
  "mac3": "",
  "mac4": "",
  "mac5": "",
  "mac6": "",
  "mac7": "",
  "moving": false,
  "rssi1": -106,
  "rssi2": -75,
  "rssi3": 0,
  "rssi4": 0,
  "rssi5": 0,
  "rssi6": 0,
  "rssi7": 0,
  "timestamp": "2089-03-04T03:37:46Z"
}
```

🚩 The issue lies with the timestamp, which appears to be unusual. The received timestamp indicates a date in the future (March 4, 2089). I would appreciate it if we could investigate this unexpected behavior further.

Thank you for your attention to this matter. Let's ensure the accuracy and proper functioning of our port configurations. 🙏

**Comment by michaelbeutler** [comment]
@havar-miromico thanks... seems to be fixed now.

**Test payload:** `000166c4a5ba00e0286d8aabfcb1e0286d8a9478c2ec6c9a74b58fad726c9a74b58dadf0b0140c96bbd0`

**Output:**
```json
{
    "bufferLevel": 1,
    "mac1": "286d8aabfcb1",
    "mac2": "286d8a9478c2",
    "mac3": "6c9a74b58fad",
    "mac4": "6c9a74b58dad",
    "mac5": "b0140c96bbd0",
    "mac6": "",
    "mac7": "",
    "moving": false,
    "rssi1": -32,
    "rssi2": -20,
    "rssi3": 114,
    "rssi4": -16,
    "rssi5": 0,
    "rssi6": 0,
    "rssi7": 0,
    "timestamp": "2024-08-20T14:18:34Z"
}
```

File: pkg/decoder/tagsl/v1/port105.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package tagsl
+
+import "time"
+
+// +-------+------+-------------------------------------------+-----------+
+// | Byte  | Size | Description                               | Format    |
+// +-------+------+-------------------------------------------+-----------+
+// | 0     | 2    | Buffer level                              | uint8     |
```
Comment (havar-miromico) [inline pkg/decoder/tagsl/v1/port105.go:8]: Byte column has some errors in there, also for the other ports e.g. buffer level 0-1, unix timestamp 2-5 etc.

**Comment by havar-miromico** [comment]
It would make sense to dissects the status byte +moving bit to separate variables.

**Comment by michaelbeutler** [comment]
> It would make sense to dissects the status byte +moving bit to separate variables.

I need more details… according to the docs, it's "Status[6:2] + Moving flag[0] (moving = 1)".

**Pull request merged**